error when runnning make gen-project

when running the `make gen-project` command in a fresh install
```
make gen-project
uv run gen-project --config-file config.yaml src/lambda_ber_schema/schema/lambda_ber_schema.yaml -d assets
```
I get the following error:
```
Traceback (most recent call last):
  File "/sc_workflow/lambda-ber-schema/.venv/bin/gen-project", line 10, in <module>
    sys.exit(cli())
             ~~~^^
  File "/sc_workflow/lambda-ber-schema/.venv/lib/python3.14/site-packages/click/core.py", line 1161, in __call__
    return self.main(*args, **kwargs)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "sc_workflow/lambda-ber-schema/.venv/lib/python3.14/site-packages/click/core.py", line 1082, in main
    rv = self.invoke(ctx)
  File "/sc_workflow/lambda-ber-schema/.venv/lib/python3.14/site-packages/click/core.py", line 1443, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/sc_workflow/lambda-ber-schema/.venv/lib/python3.14/site-packages/click/core.py", line 788, in invoke
    return __callback(*args, **kwargs)
  File "/sc_workflow/lambda-ber-schema/.venv/lib/python3.14/site-packages/linkml/generators/projectgen.py", line 256, in cli
    gen.generate(yamlfile, project_config)
    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/sc_workflow/lambda-ber-schema/.venv/lib/python3.14/site-packages/linkml/generators/projectgen.py", line 137, in generate
    gen = gen_cls(local_path, **all_gen_args)
  File "<string>", line 24, in __init__
  File "/sc_workflow/lambda-ber-schema/.venv/lib/python3.14/site-packages/linkml/generators/shexgen.py", line 42, in <lambda>
    shex: Schema = field(default_factory=lambda: Schema())  # ShEx Schema being generated
                                                 ~~~~~~^^
  File "sc_workflow/lambda-ber-schema/.venv/lib/python3.14/site-packages/ShExJSG/SchemaWithContext.py", line 14, in __init__
    super().__init__(imports, startActs, start, shapes, **_kwargs)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/sc_workflow/lambda-ber-schema/.venv/lib/python3.14/site-packages/ShExJSG/ShExJ.py", line 514, in __init__
    setattr(self, '@context', _kwargs.get('@context', None))
    ~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/sc_workflow/lambda-ber-schema/.venv/lib/python3.14/site-packages/pyjsg/jsglib/jsg_object.py", line 75, in __setattr__
    self[key] = self._map_jsg_type(key, value, vtype)
                ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "/sc_workflow/lambda-ber-schema/.venv/lib/python3.14/site-packages/pyjsg/jsglib/jsg_object.py", line 210, in _map_jsg_type
    elif conforms(element, typ, self._context.NAMESPACE):
         ~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/sc_workflow/lambda-ber-schema/.venv/lib/python3.14/site-packages/pyjsg/jsglib/conformance.py", line 40, in conforms
    return element_conforms(element, etype)
  File "/sc_workflow/lambda-ber-schema/.venv/lib/python3.14/site-packages/pyjsg/jsglib/conformance.py", line 16, in element_conforms
    if (element is None or element is Empty) and issubclass(etype, type(None)):
                                                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
TypeError: issubclass() arg 1 must be a class
make: *** [Makefile:11: gen-project] Error 1 
```

--- a/config.yaml
+++ b/config.yaml
@@ -7,6 +7,9 @@
 # Note: markdown docs are generated via gen-doc to docs/elements/ (in Makefile gendoc target)
 excludes:
   - markdown
+  # ShEx generation currently fails under Python 3.14 due to a pyjsg/ShExJSG
+  # compatibility issue during gen-project initialization.
+  - shex
 
 # Alternatively only specify the generators to run
 # (comment out the above "excludes" section)

